lib/banks/nubank.ex

```diff
@@ -0,0 +1,51 @@
+defmodule BRAN.Banks.Nubank do
+  import BRAN.Banks.Util
+
+  @moduledoc """
+    Documentation `BRAN.Banks.Nubank`
+  """
+
+  @doc """
+  Returns a tuple, after checking if the combination of agency_number, account_number and digit is valid
+
+  ##Examples
+    iex> BRAN.Banks.Nubank.validate([0,0,0,1], [5,2,1,6,1,2,5], 0)
+    :true
```

Comment:
O exemplo não condiz com o retorno da função

```suggestion
    {:ok, :valid}
```